src/server/report/icons.js

```diff
@@ -197,3 +197,15 @@ export const ICON_BROWSERS = {
   <path d="m12.967 50.59 23.416-15.752L53.2 12.173z" opacity=".243"/>
 </svg>`
 };
+
+export const ICON_BYTES = svg`
```

Comment:
Feel free to take a stab at a simpler icon, I didn't want to waste anymore time on it lol

Reply:
Haha I like this one!